tests/snuba/api/endpoints/test_organization_events_ourlogs.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime, timezone
+
+import pytest
+
+from tests.snuba.api.endpoints.test_organization_events import OrganizationEventsEndpointTestBase
+
+
+class OrganizationEventsOurLogsEndpointTest(OrganizationEventsEndpointTestBase):
+    dataset = "ourlogs"
+
+    def do_request(self, query, features=None, **kwargs):
+        query["useRpc"] = "1"
+        return super().do_request(query, features, **kwargs)
+
+    def setUp(self):
+        super().setUp()
+        self.features = {
+            "organizations:ourlogs-enabled": True,
+        }
+
+    @pytest.mark.querybuilder
+    def test_simple(self):
+        logs = [
+            self.create_ourlog(
+                {"body": "foo"},
+                timestamp=self.ten_mins_ago,
+            ),
+            self.create_ourlog(
+                {"body": "bar"},
+                timestamp=self.nine_mins_ago,
+            ),
+        ]
+        self.store_ourlogs(logs)
+        response = self.do_request(
+            {
+                "field": ["log.body"],
+                "query": "",
+                "orderby": "log.body",
+                "project": self.project.id,
+                "dataset": self.dataset,
+            }
+        )
+        assert response.status_code == 200, response.content
+        data = response.data["data"]
+        meta = response.data["meta"]
+        assert len(data) == 2
+        assert data == [
+            {"log.body": "bar"},
+            {"log.body": "foo"},
+        ]
+        assert meta["dataset"] == self.dataset
+
+    @pytest.mark.querybuilder
+    def test_timestamp_order(self):
+        logs = [
+            self.create_ourlog(
+                {"body": "bar"},
+                timestamp=self.ten_mins_ago,
+            ),
+            self.create_ourlog(
+                {"body": "foo"},
+                timestamp=self.nine_mins_ago,
+            ),
+        ]
+        self.store_ourlogs(logs)
+        response = self.do_request(
+            {
+                "field": ["log.body", "timestamp"],
+                "query": "",
+                "orderby": "timestamp",
+                "project": self.project.id,
+                "dataset": self.dataset,
+            }
+        )
```

Comment:
@wmak it works on local so the duplication must be happening elsewhere

Reply:
_weird_ thank you for writing the tests anyways